### ISSUE TITLE
training is not progressed at all using custom data, but finetuning is ok

### ISSUE THREAD

**Issue by yustain** [BVLC/caffe#1491]
I trained custom data using bvlc_reference_model.
(custom data has nearly 1500 class. each class has 1000 training samples (256x256).)
However, training was not progressed as below graph.

![train_full](https://cloud.githubusercontent.com/assets/2835392/5213109/3d832e60-764a-11e4-8553-f3bdae4a6a7f.png)

I fixed batch_size as 128 (gpu memory is 2gb.)
Then, I changed bias_filter value : 0.01 to 2 (I tried -1 ,but it was the same result) as #401
and  base_lr between 0.0005 to 10, but nothing was changed.
(Actually, when I set base_lr over 1, loss was diverged to NaN, and below 1 training was not progressed.)

I trained over 100,000 iterations using several parameter combinations, but nothing was happened.
Data was shuffled when constructing leveldb, and I checked data-label pair several times.

Do I have to tune other parameters? 
I'm running caffe on Ubuntu14.04LTS and cuda6.5.

One weird thing is training was well using finetuning setting just as guide.  
(Surely, it was the same data.)

![train_finetuning](https://cloud.githubusercontent.com/assets/2835392/5213248/e94510a4-764c-11e4-9515-0bd03709ccbf.png)
.

**Comment by wangdelp** [comment]
@sguada Hi, Sguada. I found that sometimes using Xavier init may result in the objective not converging, while using Gaussian init could get good performance. Is Xavier init really better than Gaussian init, when to use which according to your experience? 
Is it better to init bias as 0.2 instead of 0?
Thank you.
